frontend/ui-picasso/app/components/Atom/TokenAsset.tsx

```diff
@@ -17,7 +17,7 @@ export const TokenAsset: React.FC<TokenAssetProps> = ({
   const token = getToken(tokenId);
   return (
     <BaseAsset
-      icon={icon || token.icon}
+      icon={icon || token?.icon}
```

Comment:
BaseAsset should not be rendered if there is no icon. 

so `icon || token?.icon` might return an undefined value. 

it is best to check this before rendering the component.